gpt3/pretrain_345m.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```

Comment:
Would you please add Microsoft trademark?

Reply:
Done.